include/mbedtls/md.h

```diff
@@ -49,6 +49,12 @@ typedef enum {
     MBEDTLS_MD_RIPEMD160,
 } mbedtls_md_type_t;
 
+typedef enum {
+    MBEDTLS_SIG_RSA=0,
+    MBEDTLS_SIG_ECDSA,
+    MBEDTLS_SIG_NONE,
+} mbedtls_sig_type_t;
```

Comment:
@gilles-peskine-arm Yes that's another more pragmatic reason, but also conceptually I think it shouldn't be used.